### PR TITLE
Feat card

### PR DESCRIPTION
**Pull request by ankit9015**
Feature card includes:
1. Cards with badges
2. Cards with dismiss
3. Cards with text overlay
4. Text only cards
5. Vertical/Horizontal cards
6. Cards with shadow

cards

### CODE REVIEW

File: components/cards/card.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    color: var(--black);
+    padding: 0.1rem 0.35rem;;
+}
+
+.card-dismiss {
+    position: absolute;
+    top: 0.5rem;
+    right: 0.5rem;
+    color: black;
+    font-size: 20px;
```
Comment (sushil-mod) [inline components/cards/card.css]: use rem if possible

File: components/cards/card.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    font-weight: 900;
+}
+
+
+/* ------ card with shadow --------- */
+
+.card-shadow {
+    box-shadow: 5px 5px 2px 1px rgb(189, 185, 185);
+}
+
```
Comment (sushil-mod) [inline components/cards/card.css:102]: if possible use color variable

**Review by sushil-mod — APPROVED** [review]
go through comment , then you can merge !

File: components/cards/card.css (hunk 1 above)
Comment (ankit9015) [reply components/cards/card.css]: Changes done

File: components/cards/card.css (hunk 2 above)
Comment (ankit9015) [reply components/cards/card.css:102]: Changes done

File: components/cards/card.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    max-width: 300px;   
+}
+
+.card-horizontal {
+    max-width: 700px;
+    flex-direction: row;
+}
+
+
+.card-img {
```
Comment (Kothalkarpriya) [inline components/cards/card.css:39]: remove empty selectors

File: components/cards/card.css [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+.card-dismiss {
+    position: absolute;
+    top: 0.5rem;
+    right: 0.5rem;
+    color: var(--black);
+    font-size: 2rem;
+}
+
+
```
Comment (Kothalkarpriya) [inline components/cards/card.css:80]: remove extra spaces

File: components/cards/cards.html [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    
+    <link rel="stylesheet" href="./card.css">
+    <link rel="stylesheet" href="../components.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="cards-container-flex">
+
+        <div class="card-vertical">
+            <img class="card-img" src=https://picsum.photos/200/200 alt="">
```
Comment (Kothalkarpriya) [inline components/cards/cards.html]: for image tag do use / before closing it, as it is self closing tag

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Look into my comments

**Pull request merged**